server/public/model/metrics.go

```diff
@@ -86,20 +88,20 @@ func (r *PerformanceReport) IsValid() error {
 
 	reportVersion, err := semver.ParseTolerant(r.Version)
 	if err != nil {
-		return err
+		return fmt.Errorf("could not parse semver version: %s, %v", r.Version, err)
```

Comment:
nit: Should we `%w` the `err`, or are there reasons to avoid doing so?

Reply:
Probably `%w` is better to keep things extensible.